src/handlers/movies/movieById.ts

```diff
@@ -0,0 +1,36 @@
+import { APIGatewayProxyHandler } from 'aws-lambda';
+import { notFoundResponse, serverErrorResponse } from '../../utils/api/apiResponses';
+import getMovieById from '../../models/Movie/getById';
+import { CONTENTFUL_INCLUDE } from '../../types/contentful';
+import { isCustomContentfulError } from '../../utils/api/utils';
+
+export const handler: APIGatewayProxyHandler = async (event) => {
+	const movieId = event?.pathParameters?.id;
+
+	if (!movieId) {
+		return notFoundResponse;
+	}
```

Comment:
If we change the path to `/movies/{id}` this verification won't be necessary as it will fall back to the movies search path.